netbox/utilities/counters.py

```diff
@@ -62,6 +62,11 @@ def post_save_receiver(sender, instance, created, **kwargs):
             update_counter(parent_model, new_pk, counter_name, 1)
 
 
+def pre_delete_receiver(sender, instance, origin, **kwargs):
+    if not type(instance).objects.filter(pk=instance.pk).exists():
```

Comment:
Would it be enough to check if `instance.pk is None`? `pk` should be nullified for any instance that's been deleted, but I'm not sure if we might encounter duplicate instances.

Reply:
That is sort of the heart of the problem, it gets sent twice and instance.pk is not None.